Library/Formula/rust.rb

```diff
@@ -44,7 +44,6 @@ def install
     args = ["--prefix=#{prefix}"]
     args << "--disable-rpath" if build.head?
     args << "--enable-clang" if ENV.compiler == :clang
-    args << "--release-channel=beta" unless build.head?
```

Comment:
you may or may not want to re-add a `--release-channel=nightly" if build.head?` too